### PR TITLE
Poc cases

### PR DESCRIPTION
**Pull request by jonathangriff**

### CODE REVIEW

File: src/components/Case.vue [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  }
+
+  &:hover {
+    .case__nextIcon {
+      transform: translate(0.25rem, -50%);
+      transition: ease-in-out transform 0.3s;
+    }
+  }
+
+  &--1 {
```
Comment (lam218) [inline src/components/Case.vue]: Not to necessarily a bad point, just might be nice to change to --success, --error etc. just to make styling really explicit

File: src/components/Cases.vue [hunk 2]
```diff
@@ -0,0 +1,284 @@
+<template>
+  <div class="cases">
+    <div class="cases__container">
+      <!-- Loading -->
+      <div v-if="isLoading">
+        Loading...
+      </div>
+      <!-- Error -->
+      <div v-if="!isLoading && error">
```
Comment (lam218) [inline src/components/Cases.vue]: can use v-else I think here?

File: src/components/Cases.vue [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+        Error message to go here
+      </div>
+      <div v-if="!isLoading && !error" class="cases__listing">
```
Comment (lam218) [inline src/components/Cases.vue]: can use v-else here I think?

File: src/components/Cases.vue [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+      showFilters: false,
+      searchTerm: '',
+      status: 0,
+    }
+  },
+  computed: {
+    filteredList() {
+      // Search name and caseNumber
+      const searchedList = this.payload.filter(item => {
+        return item.name.toLowerCase().includes(this.searchTerm.toLowerCase())
```
Comment (lam218) [inline src/components/Cases.vue]: probably need to use indexof here, instead .includes doesn't work with ie

File: src/components/Cases.vue [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+  computed: {
+    filteredList() {
+      // Search name and caseNumber
+      const searchedList = this.payload.filter(item => {
+        return item.name.toLowerCase().includes(this.searchTerm.toLowerCase()) 
+        || item.caseNumber.toLowerCase().includes(this.searchTerm.toLowerCase());
+      });
+
+      // Status filter
+      const filteredList = parseInt(this.status) !== 0
```
Comment (lam218) [inline src/components/Cases.vue]: Can you pass the status through as a number to avoid having to parse?

File: src/components/Cases.vue [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+    box-shadow: 0 0 0 1px #65858f;
+    border: 0;
+    -webkit-appearance: none;
+    line-height: normal;
+  }
+
+  .dropdown {
+    width: 100%;
+
+    .dropdown__select {
```
Comment (lam218) [inline src/components/Cases.vue]: &__select

File: src/components/ContactUs.vue [hunk 7]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="contact">
+    <div class="contact__description">
+      <h5 class="contact__title">Contact us to renew quotation {{ caseNumber }}</h5>
```
Comment (lam218) [inline src/components/ContactUs.vue]: Might want a v-cloak here?

File: src/components/Action.vue [hunk 8]
```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="action">
+    <div class="action__content">
+      <h4 class="action__title">{{ title }}</h4>
```
Comment (lam218) [inline src/components/Action.vue]: Consider v-cloak if there is a split second of handlebars?

File: src/components/ContactUs.vue [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+        </svg>
+        0800 052 0130
```
Comment (lam218) [inline src/components/ContactUs.vue]: Want to make this an anchor with tel href?

File: src/components/Filters.vue [hunk 10]
```diff
@@ -0,0 +1,186 @@
+<template>
+  <div class="filters">
+    <button v-on:click="toggleFilters" class="filters__toggle">
```
Comment (lam218) [inline src/components/Filters.vue]: want to change this to ``@click`` to keep consistency?

File: src/components/Filters.vue [hunk 11]
```diff
@@ -0,0 +1,186 @@
+<template>
+  <div class="filters">
+    <button v-on:click="toggleFilters" class="filters__toggle"> 
+      <div class="cases_toggleShowing">
+        Showing: <b> {{ statusMessage }}</b>
+        <span v-if="searchTerm"> matching <b>{{ searchTerm }}</b></span>
+      </div>
+      <div class="filters__toggleLabel">
+        <span class="filters__toggleLabelText">Search cases </span>
+        <svg class="filters__toggleIcon"  v-bind:class="{ active: showFilters }"  viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
```
Comment (lam218) [inline src/components/Filters.vue]: same with the v-bind

File: src/components/Pagination.vue [hunk 12]
```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="pagination">
+    <button @click="previousPage" class="pagination__previous">
```
Comment (lam218) [inline src/components/Pagination.vue]: add enter for previous, for accessibility

File: src/components/Filters.vue [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+          <path d="M12.4536518,8.69541757 C11.8487827,8.07880352 11.8487827,7.0790746 12.4536518,6.46246054 C13.0585208,5.84584649 14.0392073,5.84584649 14.6440763,6.46246054 L24,16.0000526 L14.6441796,25.5375395 C14.0393105,26.1541535 13.0586241,26.1541535 12.453755,25.5375395 C11.848886,24.9209254 11.848886,23.9211965 12.453755,23.3045824 L19.619151,16.0000526 L12.4536518,8.69541757 Z" fill="currentColor"></path>
+        </svg>
+        </div>
+    </button>
+    <div class="filters__filtersBody" v-bind:class="{ active: showFilters }">
+      <div class="filters__filtersContainer">
+        <div class="filters__filtersCol">
+          <input v-model="searchTermValue" class="input" />
```
Comment (lam218) [inline src/components/Filters.vue]: does this have a label or placeholder? Just thinking about accessibility

File: src/components/Tab.vue [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+<script>
+export default {
+  name: 'Tab',
+  props: {
+    title: {
+      required: true,
+      type: String,
+    },
+
```
Comment (lam218) [inline src/components/Tab.vue]: weird white space (sorry!)

File: src/components/Tabs.vue [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+      tabs: [],
+    }
+  },
+  created() {
+    this.tabs = this.$children;
+  },
+  methods: {
+    selectTab(selectedTab) {
+      this.tabs.forEach(tab => {
+        tab.isActive = (tab.tabId == selectedTab.tabId);
```
Comment (lam218) [inline src/components/Tabs.vue]: can we have a === here?

File: src/components/Case.vue (hunk 1 above)
Comment (lam218) [reply src/components/Case.vue]: also, not necessarily requiring a change here, you can just resolve if you want

Comment (jonathangriff) [reply src/components/Case.vue]: Added comments above the modifier

File: src/components/Cases.vue (hunk 3 above)
Comment (jonathangriff) [reply src/components/Cases.vue]: Error should be true.

File: src/components/Filters.vue (hunk 13 above)
Comment (lam218) [reply src/components/Filters.vue]: could we have for on the label and id on the input?

**Pull request merged**